spec/lib/levenshtein_spec.rb

```diff
@@ -48,6 +49,15 @@
     specify{ Vladlev.distance('*&^%$+_=-)(*&^%$#@!~123456789', '*&^%$+_=-)(*&^%$#@!~').should equal(9) }
     specify{ Vladlev.distance('*&^%$+_=-)(*&^%$#@!~', "").should equal(20) }
   end
+
+  describe "normalized distance" do
+    specify{ expect(Vladlev.distance("cur", "cut", 10)).to eq(1) }
+    specify{ expect(Vladlev.get_normalized_distance("hello", "hello")).to eq(0.0) }
```

Comment:
Should switch this unit test call to `distance()` to `get_normalized_distance()`.